subjects/devops/master-the-ls/README.md

```diff
@@ -0,0 +1,51 @@
+## Master the ls
+
+### Instructions
+
+Put in a file `master-the-ls`, the command line that will:
```

Comment:
i propose to create a script called: master-the-ls.sh

Reply:
done